map/create-and-save-map/src/main/java/com/esri/samples/create_and_save_map/CreateAndSaveMapController.java

```diff
@@ -60,7 +62,7 @@
   @FXML
   private ComboBox<PortalFolder> folderList;
   @FXML
-  private ListView<Basemap> basemapList;
+  private ListView<BasemapStyle> basemapStyleList;
```

Comment:
Maybe List View here as well? I see the other variables could also be named a bit better, so up to you if you want to include it in this PR or not
```suggestion
  private ListView<BasemapStyle> basemapStyleListView;
```